include/solana.hpp

```diff
@@ -111,6 +111,31 @@ struct TransactionReturnData {
  */
 void from_json(const json &j, TransactionReturnData &data);
 
+/**
+ * The level of commitment desired when querying state
+ */
+enum Commitment {
```

Comment:
Small detail but should start using enum class instead of enums. Does not change a lot in our case.

Reply:
sure. Will keep in mind